src/Http/RequestManager.php

```diff
@@ -127,7 +129,10 @@ protected function createHttpRequestFromMatejRequest(Request $request): RequestI
             ->createRequest(
                 $request->getMethod(),
                 $uri,
-                ['Content-Type' => 'application/json'],
+                [
+                    'Content-Type' => 'application/json',
```

Comment:
ehmm... maybe you should use this awesome library https://github.com/lmc-eu/http-constants 😄

Reply:
@davidkmenta Yes, I heard about its awesomeness as well - but its IMO more useful when building larger app where you use those headers more frequently. As for the library, it won't pay for itself (for now) - and I'm trying to not expand the dependencies footprint of this library if not necessary.